json/src/spec/params.rs

```diff
@@ -117,6 +115,11 @@ pub struct Params {
 	/// Transaction permission contract address.
 	#[serde(rename="transactionPermissionContract")]
 	pub transaction_permission_contract: Option<Address>,
+	/// Wasm support flag
+	pub wasm: Option<bool>,
```

Comment:
Is it still needed? What if it's set to `wasm: true && wasm_activiation_transition: 500` is it activated right away or at block `500`?

IMHO we should keep spec files clean (even if it requires a migration of existing spec files), especially given `wasm` was always experimental to date.

Reply:
no, not needed, probably should mention that it requires to change in json spec to replace

`wasm: true` with `wasmActivationTransition: "0x0"`